modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/cache/CacheSubcommands.java

```diff
@@ -88,6 +89,11 @@ public enum CacheSubcommands {
      */
     INDEX_FORCE_REBUILD("indexes_force_rebuild", IndexForceRebuildCommandArg.class, new CacheIndexesForceRebuild()),
 
+    /**
+     * Index rebuild via the maintenance mode.
+     */
+    INDEX_REBUILD("indexes_rebuild", IndexRebuildCommandArg.class, new CacheIndexesRebuild()),
```

Comment:
**indexes_rebuild** doesn't sound to me like scheduling an index rebuild, but how to perform an index rebuild, I suggest calling it something else: **schedule_indexes_rebuild**